### ISSUE TITLE
.gitignore 설정

### ISSUE THREAD

**Issue by yewonkang00** [COALA-Web/COALA#8]
node_modules가 .gitignore 파일에 포함되어 있는데 node_modules 파일이 너무 커서 설정하신건가요??
그리고 .gitignore에 포함되어야 할 파일이 또 있을까요?

**Comment by YuriKwon** [comment]
네. 차지하는 용량이 너무 크고 `package.json`으로 대체 가능하기 때문에 `.gitignore` 에 추가했습니다. 
그 외에는 통상적으로 log 관련 파일도 `.gitignore`에 추가한다고 합니다.
